src/menu/stateful-container.js

```diff
@@ -149,11 +154,65 @@ export default class MenuStatefulContainer extends React.Component<
         }
         this.handleEnterKey(event);
         break;
+      default:
+        if (this.props.typeAhead) this.handleAlphaDown(event);
+        break;
+    }
+  };
+  handleAlphaDown = (event: KeyboardEvent) => {
+    const rootRef = this.props.rootRef ? this.props.rootRef : this.rootRef;
+    const prevIndex = this.state.highlightedIndex;
+
+    this.typeAheadChars += event.key;
+    this.typeAheadTimeOut = setTimeout(() => {
```

Comment:
It looks like there is no `clearTimeout` included in this diff. Check the earlier example I wrote on how to handle this feature. It should `clearTimeout` in the keydown event listener. Currently if my text input takes longer than `500ms` I would not be able to search for longer strings

Reply:
Thanks！